### PR TITLE
Create lgtm config

### PR DESCRIPTION
**Pull request by paulfantom**
Skeleton for exclusion of files and LGTM tests/queries.

Sparked in #4203

### CODE REVIEW

File: .lgtm.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# docs: https://lgtm.com/help/lgtm/lgtm.yml-configuration-file
+
+path_classifiers:
+  test:
+    - exclude: python.d/python_modules/third_party/*
+    - exclude: python.d/python_modules/urllib3/*
+    - exclude: python.d/python_modules/pyyaml2/*
+    - exclude: python.d/python_modules/pyyaml3/*
+queries:
+  - exclude: cpp/command-line-injection
```
Comment (ktsaou) [inline .lgtm.yml]: don't remove it yet please.
we need this active to file the bug.

Comment (ktsaou) [reply .lgtm.yml]: applied work-around with https://github.com/firehol/netdata/pull/4218
as suggested by https://github.com/Semmle/ql/issues/199#issuecomment-421996400

So, @paulfantom you can remove the exclusion for `cpp/command-line-injection`

**Comment by paulfantom** [comment]
@ktsaou Do we also need to exclude `cpp/fixme-comment`. LGTM check seems to be operating only on PR diff, so it is not showing an error is we don't introduce any new errors. So by not excluding `cpp/fixme-comment` we will have a place which shows what should be fixed in the future. I believe that's why this check was introduced.

**Comment by ktsaou** [comment]
I think it influences the grade it gives to a project. So, forget about A+ with these in place.
Check this: https://lgtm.com/projects/g/firehol/netdata/context:cpp

Let's do this: LGTM needs some time to process a repo and calculate its grade. The last PRs merged today, are still not processed. So, I suggest to comment out the exclusion and merge this.

Currently, the `C` and `js` code is completely clean. Only `FIXME` alerts on LGTM.

Once we will know our grade, we will decide. Although, to be fare among all projects, this exclusion should not influence the project's grade, or even better `FIXME` comments should not influence the grade in the first place. Let's see what they decided...

**Comment by ktsaou** [comment]
btw, I can't figure out what the size of the circle of each project is on this page: https://lgtm.com/projects/g/firehol/netdata/context:cpp

It is not code size, or stars, or contributors. I can't figure out what it is... Probably PR/commit rate?

**Comment by paulfantom** [comment]
We have some problems which should be fixed and we know about them, that's why there are FIXME comments. For me this means that we shouldn't have A+ grade as grading should be done on current state, not on our aspirations or the fact of us knowing that we are doing sth not in the best way, but we don't know any better yet.

**Comment by paulfantom** [comment]
Here is about circle size: https://help.semmle.com/lgtm-enterprise/user/help/comparing-projects-contributors.html

**Comment by ktsaou** [comment]
Activity—in lines changed, over the revisions analyzed (circle size)
haha...

**Comment by ktsaou** [comment]
> Activity—in lines changed, over the revisions analyzed (circle size)

This actually promotes projects that are not so stable... Stable code means fewer lines changed.

**Comment by ktsaou** [comment]
ok, could you please remove all query exclusions?

**Comment by paulfantom** [comment]
We probably need to add more path classifiers, but this can be also done later.

**Comment by ktsaou** [comment]
Thank you!

**Pull request merged**